### PR TITLE
AB#5019

### PR DESCRIPTION
**Pull request by sachin755**
### Proposed changes

*
*

### Related issues

*
*

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by johnjdl** [comment]
@r-joyce review and merge when you have a moment.

**Comment by johnjdl** [comment]
@r-joyce hold off on this one, still some queries needed to implement into this one.

File: opencti-platform/opencti-front/src/relay/environmentDarkLight.js
Comment (r-joyce) [inline opencti-platform/opencti-front/src/relay/environmentDarkLight.js]: I'm guessing this was a mistake

**Review by r-joyce — CHANGES_REQUESTED** [review]

**Comment by johnjdl** [comment]
@r-joyce go ahead and merge this along with 5014 and 5017.

**Comment by johnjdl** [comment]
@r-joyce nevermind, hold on merging 5014, 5017, and 5019 for now.

**Comment by johnjdl** [comment]
@r-joyce please review and merge these first today.

5014 - Data Entity: Parties
*5019 - Data Entity: Locations
5017 - Data Entity: Assessment Platform
5195 - Disable merge button

File: opencti-platform/opencti-front/src/private/components/dataEntities/data/locations/EntityLocationCard.js
Comment (r-joyce) [inline opencti-platform/opencti-front/src/private/components/dataEntities/data/locations/EntityLocationCard.js]: Lorem Ipsum

**Review by r-joyce — APPROVED** [review]
Looks good 👍

**Pull request merged**
